testsuite/integration-arquillian/pom.xml

```diff
@@ -534,12 +530,12 @@
             </properties>
         </profile>
         <profile>
-            <id>db-allocator-db-oracle11g</id>
+            <id>db-allocator-db-oracle12cR1RAC</id>
             <properties>
                 <jdbc.mvn.groupId>com.oracle</jdbc.mvn.groupId>
                 <jdbc.mvn.artifactId>ojdbc7</jdbc.mvn.artifactId>
                 <jdbc.mvn.version>12.1.0</jdbc.mvn.version>
```

Comment:
I'd recommend to use https://mvnrepository.com/artifact/com.oracle/ojdbc8/12.2.0.1

Reply:
Changed to this version as we discussed